### PR TITLE
Feat: Migrate mobile common components to `@dzcode.io/ui-mobile`

### LINKED ISSUE #494: migrate mobile common components to `@dzcode.io/ui-mobile`
migrating mobile common components to `@dzcode.io/ui-mobile`

- [x] AppBar
- [x] DrawerContent
- [x] ErrorBoundary
- [x] Loading
- [x] TryAgain

### PR DESCRIPTION
**Pull request by omdxp**
# Description

This PR is dedicated to migrate the rest of common components to `@dzcode.io/ui-mobile` package.

Resolves #494 

## Type of change

- New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: mobile/src/screens/projects/card-item/index.tsx [hunk 1]
```diff
@@ -18,6 +18,8 @@ const CardItem: FC<CardItemProps> = ({
   return (
     <Card style={cardStyles.mainView}>
       <Card.Cover
+        height
+        width
```
Comment (ZibanPirate) [inline mobile/src/screens/projects/card-item/index.tsx:22]: what are these?

if they are booleans, then we should pass them like this:
```suggestion
        height={true}
        width={true}
```

I will make a note to enforce this using [react/jsx-boolean-value](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md)

**Review by ZibanPirate — CHANGES_REQUESTED** [review]
Also, some of the PNG assets are not used, can we remove those?

File: mobile/src/screens/projects/card-item/index.tsx (hunk 1 above)
Comment (omdxp) [reply mobile/src/screens/projects/card-item/index.tsx:22]: They're actually unknowns but they are not optionals thus some ts errors were thrown from this component, do you want to enforce them to be equal to true?

**Review by ZibanPirate — APPROVED** [review]

File: mobile/src/screens/projects/card-item/index.tsx (hunk 1 above)
Comment (ZibanPirate) [reply mobile/src/screens/projects/card-item/index.tsx]: that will do, we will later update `eslint` to enforce that rule

**Pull request merged**